src/plugins/maxlength/main.rs

```diff
@@ -0,0 +1,42 @@
+use overrider::*;
+
+use crate::drw::Drw;
+use crate::item::Item;
+use crate::result::*;
+use crate::clapflags::CLAP_FLAGS;
+
+#[override_flag(flag = maxlength)]
+impl Drw {
+    pub fn postprocess_matches(&mut self, mut current_matches: Vec<Item>) -> CompResult<Vec<Item>> {
+	let max_length_str_option = CLAP_FLAGS.value_of( "maxlength" );
+
+	match max_length_str_option
+	{
+		None => {
+			Err(Die::Stderr("Please specificy max length".to_owned()))
```

Comment:
```rust
Err(Die::Stderr("Please specificy max length".to_owned()))
```
can be shortened to
```rust
Die::stderr("Please specificy max length".to_owned())
```
This also applies to lines 23 and 31 (with Die::stdout on 31). However, these are stylistic choices. I'd merge even if you left these as is.

Reply:
I let it be because I am returning both `Ok` and `Err` branches so I cannot have an Err in the enclosing brackets for the shorter version.